extensions/funqy/funqy-google-cloud-functions/runtime/src/main/java/io/quarkus/funqy/gcp/functions/FunqyHttpFunction.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkus.funqy.gcp.functions;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+import com.google.cloud.functions.HttpFunction;
+import com.google.cloud.functions.HttpRequest;
+import com.google.cloud.functions.HttpResponse;
+
+import io.quarkus.runtime.Application;
+
+public class FunqyHttpFunction implements HttpFunction {
```

Comment:
We is there a specific need for Funqy + HTTP + GCP?  Isn't this handled by a more generic function?

Reply:
GCF have 3 distinct function interface, one for HttpFunction and two for BackgroundFunction. So I use two different implementation.
We can mix the two function inside he same implementation if you think it's easier for the user to use and easier for us to maintain.

Reply:
If I compare whith what you provides for Amazon, Funqy HTTP for GCF should be using both `funqy-http` and `google-cloud-http`.
If it's the case, I need to test it and, if working, just provides documentation.

Reply:
Yes exactly:  funqy-http + google-cloud-http.  No need for a funqy-gcf-http layer.

Also, what about my other question?  If you have funqy integrated with gcf http, why would you want background implementation?  I can understand pub/sub, but what does background provide?

Reply:
OK, I'll remove it and document the fact that funqy-http will work with google-cloud-functions-http.

Background is the generic term for not-http function, it is triggered by any event that Google Cloud support (pubsub, storage, firestore, ...).
So this is why we need Funqy for.
I give examples for PubSub and Storage, on the example I show how you can trigger them via HTTP for testing purpose (via the local invoker) but on GCP this will not be triggered via HTTP so we need Funqy for it.

Reply:
HttpFunction binding removed, we can know call only background function via Funqy (so only respond to Cloud events, not to HTTP trigger).